app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -395,6 +395,19 @@ def save_container_volumes_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.container_volumes, hashes, :name)
   end
 
+  def save_custom_attributes_inventory(entity, hashes, target = nil)
```

Comment:
Yes. I will rephrase: What a cool function! Lets rewrite `save_custom_attribute_attribute_inventory` with it in another PR!